### PR TITLE
Add information on Lyon hackathon

### PR DESCRIPTION
**Pull request by razvand**
Hackathon will take place on the weekend of May 14-15, 2022, at ENS de Lyon.

### CODE REVIEW

File: content/en/community/hackathons/2022-05-lyon/_index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+* 14:00-15:15: Running (Complex) Application in Unikraft
+* 15:15-17:00: FlexOS
+
+**Sunday, May 15, 2022**
+* 09:00-09:15: Overview of hackathon challenges
+* 09:15-10:30: Tutorial on porting applications
+* 10:30-13:00: Work on hackathon challenges
+* 13:00-14:00: Lunch
+* 14:00-17:00: Work on hackathon challenges
+* 17:00-17:30: Results, final remarks
```
Comment (nderjung) [inline content/en/community/hackathons/2022-05-lyon/_index.md:49]: I'd put this into a markdown table format.

File: content/en/community/hackathons/_index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Hackathons
+date: 2022-03-31T03:40:00+02:00
+draft: false
+collapsible: true
```
Comment (nderjung) [inline content/en/community/hackathons/_index.md:5]: Can you add a `description:` attribute with a short sentence summarising this page?

File: content/en/community/hackathons/_index.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+## Unikraft Hackathons
+
+We organize hackathons to help people make their first steps in Unikraft and learn more about operating systems, virtualizaion and unikernel technology.
+
+We aim to have hackathons as face-to-face events for maximum support from the Unikraft community members acting as teaching assistants (TAs) and support crew.
+In any case (face-to-face or strict only), the hackathon will also take place on [Unikraft Discord](https://bit.ly/UnikraftDiscord) with talks / presentations, breakout rooms and virtual support from Unikraft TAs.
+
+If you want to host a Unikraft hackathon, please get in touch with us via [Discord](https://bit.ly/UnikraftDiscord).
```
Comment (nderjung) [inline content/en/community/hackathons/_index.md:15]:
```diff
- If you want to host a Unikraft hackathon, please get in touch with us via [Discord](https://bit.ly/UnikraftDiscord).
+ If you want to host a Unikraft hackathon, please [get in touch with us](https://unikraft.org/community/contacts/).
```

**Review by nderjung — COMMENTED** [review]
Thanks Razvan! Just a few suggestions in-line.  I've also created a banner to share and add to the Lyon hackathon, i'll push this to your branch.

**Comment by razvand** [comment]
@nderjung , solved. Please do another pass.

File: content/en/community/hackathons/2022-05-lyon/_index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+The hackathon will take place at the ENS de Lyon.
+The hosts of the hackathon are:
+* Alain Tchana, Associate Professor at ENS de Lyon
+* Renaud Lachaize, Assistant Professor at UGA in Grenoble
+* Djob Mvondo, Assistant Professor at Rennes University
+
+The Unikraft community will be present on-site for the hackathon:
+* Pierre Olivier
+* Hugo Lefeuvre
+* Răzvan Deaconescu
```
Comment (nderjung) [inline content/en/community/hackathons/2022-05-lyon/_index.md:31]: Maybe you want to add emails?

**Review by nderjung — COMMENTED** [review]
Overall it's good now! Thanks.  One suggestion but happy to accept as is depending on your opinion, see below:

File: content/en/community/hackathons/2022-05-lyon/_index.md (hunk 4 above)
Comment (razvand) [reply content/en/community/hackathons/2022-05-lyon/_index.md]: @nderjung , done.

**Review by StefanJum — APPROVED** [review]
Looks good!

Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Review by nderjung — APPROVED** [review]
Thanks!

Approved-by: Alexander Jung <alex@unikraft.org>

**Pull request merged**